src/TauBounds.cpp

```diff
@@ -69,13 +134,15 @@ void printTauBounds(int iteration, FILE *file) {
   fprintf(file, "%d", iteration);
   for (int pop = 0; pop < dataSetup.popTree->numPops; pop++) {
     if (isLeafPop(pop)) continue;
-    fprintf(file, "\t%.40f\t%.40f", tau_bounds[pop], dataSetup.popTree->pops[pop]->age);
+    fprintf(file, "\t%.40f\t%.40f\t%.40f", tau_ubounds[pop], tau_lbounds[pop], dataSetup.popTree->pops[pop]->age);
```

Comment:
the `tau_bounds.tsv` output file now contains another column per comb. btw, @gphocs-dev , do we also need to calculate taubounds for clades? let's discuss!